crates/sui-bridge/src/types.rs

```diff
@@ -102,7 +108,7 @@ pub enum TokenId {
     USDT = 4,
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
```

Comment:
For the TokenId enum above, do we want to make it more extensible?

Reply:
how to make it more extensible?

Reply:
For TokenID, does the below work?
```
struct TokenID {
    name: String,
    native_chain: String,
}
```
Basically I wonder if the TokenID has to be an enum.

Reply:
I see. It's made an enum so we can easily present it as an u8. We don't need the source chain info in v1 (and probably not until we support more chains)